plantuml-mode.el

```diff
@@ -479,15 +479,14 @@ Put the result into buffer BUF, selecting the window according to PREFIX:
 (defun plantuml-preview-string (prefix string)
   "Preview diagram from PlantUML sources (as STRING), using prefix (as PREFIX)
 to choose where to display it."
-  (let ((b (get-buffer plantuml-preview-buffer)))
-    (when b
-      (kill-buffer b)))
-
   (let* ((imagep (and (display-images-p)
                       (plantuml-is-image-output-p)))
          (buf (get-buffer-create plantuml-preview-buffer))
          (coding-system-for-read (and imagep 'binary))
          (coding-system-for-write (and imagep 'binary)))
+    (with-current-buffer buf
+      (image-toggle-display-text)
```

Comment:
I've two issues with this line, one technical and one philosophical: 
- technical) on a fresh Emacs (running wtih `emacs -q`), when I try to preview a plantuml-mode buffer I get `save-current-buffer: Symbol’s function definition is void: image-toggle-display-text`
- philosophical) we support both image (PNG, SVG) as well as textual (UTXT) output formats in plantuml-mode, wouldn't we need to first check if we're actually displaying an image here? Also, reusing the same buffer breaks the possiblity to switch between image and textual output (e.g. `M-x plantuml-set-output-type txt`) since the buffer is still in `image-mode`